src/app/not-found/not-found.component.html

```diff
@@ -0,0 +1,60 @@
+<!doctype html>
+<html lang="en" style="height: 100%">
+  <head>
+    <meta charset="utf-8"/>
+    <title>MifosX - 404</title>
+    <base href="/"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="mobile-web-app-capable" content="yes">
+
+    <link rel="icon" type="image/x-icon" href="favicon.ico"/>
+    <link rel="icon" type="image/png" href="assets/images/MifosX_logo.png" sizes="182x184">
+    <link rel="apple-touch-icon" type="image/png" href="apple-touch-icon.png" sizes="182x184"/>
+    <link rel="manifest" href="manifest.json"/>
+
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Mono:300" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
+  </head>
+  <body id="bg">
+    <!--[if lt IE 10]>
+      <p>
+        You are using an <strong>outdated</strong> browser.
+        Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.
+      </p>
+    <![endif]-->
+
+    <noscript>
+      <p>
+        This page requires JavaScript to work properly. Please enable JavaScript in your browser.
+      </p>
+    </noscript>
+
+    <script>
+      // This hides the address bar on mobile browsers
+      // https://developers.google.com/web/fundamentals/native-hardware/fullscreen/
+      window.addEventListener('load',function() { window.scrollTo(0, 1); });
+    </script>
+
+    <mifosx-web-app>
+      <div class="loader-wrapper">
+        <div class="bottom triangle"></div>
+        <div class="top triangle"></div>
+        <div class="left triangle"></div>
+        <div class="right triangle"></div>
+      </div>
+    </mifosx-web-app>
+
+    <div id="particles-js" class="center"></div>
+    <div class="center">
+      <h1 style="color: white; font-family: 'Raleway', sans-serif; font-size: 120pt;">404</h1>
+      <p style="color: white; font-family: 'Raleway', sans-serif; font-size: 12pt;">Something went wrong.</p>
+    </div>
+
+    <script src="http://cdn.jsdelivr.net/particles.js/2.0.0/particles.min.js"></script>
+    <script>
+      particlesJS.load("particles-js", 'particles.json');
+    </script>
+  </body>
```

Comment:
The code starting from this line till the end is not supposed to be in the component's html file. Remove it.